Comp2024/src/main/java/frc/robot/Constants.java

```diff
@@ -64,40 +64,43 @@ public class Constants
   /////////////////////////////////////////////////////////////////////////////
   public static final class Ports
   {
-    public static final String kCANCarnivore     = "canivore1";
-    public static final String kCANRio           = "rio";
+    public static final String kCANCarnivore              = "canivore1";
+    public static final String kCANRio                    = "rio";
 
     // CANivore CAN IDs - Swerve
-    public static final int    kCANID_DriveLF    = 1;
-    public static final int    kCANID_AngleLF    = 2;
-    public static final int    kCANID_CANCoderLF = 3;
+    public static final int    kCANID_DriveLF             = 1;
```

Comment:
Let's put the device type in an comment for each of these like:
kCANID_DriveLF = 1;  // Falcon 500/TalonFX
etc.

And we should use "steer" instead of "angle" for the steering motors.